nipype/interfaces/freesurfer/tests/test_preprocess.py

```diff
@@ -6,11 +6,12 @@
 import nibabel as nif
 import numpy as np
 from tempfile import mkdtemp
-from nipype.testing import assert_equal, assert_raises, skipif
+import pytest
 import nipype.interfaces.freesurfer as freesurfer
 
 
-def create_files_in_directory():
+@pytest.fixture()
+def create_files_in_directory(request):
```

Comment:
it can be, let me do it in a separate PR, I will review some other fixtures. I believe this is not the only one that is often repeated